kustomize/components/memorystore/README.md

```diff
@@ -4,28 +4,40 @@ By default the `cartservice` app is serializing the data in an in-cluster Redis
 
 ![Architecture diagram with Memorystore](/docs/img/memorystore.png)
 
-To use Memorystore (Redis), the `cartservice`'s environment variable `REDIS_ADDR` needs to be updated to point to the Memorystore (Redis) instance.
-
-To provision a Memorystore (Redis) instance you can follow the instructions [here](https://cloud.google.com/memorystore/docs/redis/creating-managing-instances).
-
-You can also find in this repository the Terraform script to provision the Memorystore (Redis) instance alongside the GKE cluster, more information [here](/terraform).
+## Provision a Memorystore (Redis) instance
 
 Important notes:
 - You can connect to a Memorystore (Redis) instance from GKE clusters that are in the same region and use the same network as your instance.
 - You cannot connect to a Memorystore (Redis) instance from a GKE cluster without VPC-native/IP aliasing enabled.
 
+To provision a Memorystore (Redis) instance you can follow the following instructions:
```

Comment:
To be consistent with Spanner, I think that's a good idea to have ready-to-use gcloud command instead of asking the end-users to search how to do that.